spec/xcpretty/formatters/formatter_spec.rb

```diff
@@ -152,5 +156,15 @@ module XCPretty
 
 #{@formatter.red(SAMPLE_EXECUTED_TESTS)})
     end
+
+    it "show unmatched lines" do
+      @formatter = Formatter.new(true, true, true)
+      @formatter.format_other(SAMPLE_UNMATCHED_LINE).should == "unmatched: " + SAMPLE_UNMATCHED_LINE
```

Comment:
Just noticed - there is already a constant called `SAMPLE_FORMAT_OTHER_UNRECOGNIZED_STRING`. You should probabaly use that.

Reply:
wow, nice catch, I scrolled the constant.rb file searching for one like this...